tests/GraphQL/FakeResolveInfo.php

```diff
@@ -2,6 +2,8 @@
 
 namespace DNADesign\Elemental\Tests\GraphQL;
 
+use ArrayObject;
+use GraphQL\Language\AST\OperationDefinitionNode;
```

Comment:
```
use ArrayObject;
use GraphQL\Language\AST\OperationDefinitionNode;
```
Required types of parameters in ResolveInfo constructor : https://github.com/webonyx/graphql-php/blob/master/src/Type/Definition/ResolveInfo.php#L123-L132